lib/tasks/fake_user_data.rake.example

```diff
@@ -0,0 +1,33 @@
+# Replace <your gmail name> with your GMail name below.
```

Comment:
I just realized that `lib/tasks/fake_user_data.rake` needs to be added to the .gitignore.

But honestly I think a better way to do this would be to put the admin gmail ID as an optional setting in local_config. This way the user doesn't have to copy and rename yet another file.

You could/should also raise an error if the config setting is not defined and this script is run.